Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM debian:stable
+
+RUN apt update -y
+
+RUN apt install -y alien
+
+WORKDIR /tmp
+
+RUN mkdir img
+
+RUN mkdir src
+
+COPY src/ src
+
+COPY zerotier-gui.desktop .
+
+COPY make_deb.sh .
+
+COPY README.md .
+
+COPY LICENSE .
+
+RUN chmod +x make_deb.sh
+
+RUN ./make_deb.sh
+
+RUN alien -r ZeroTier-GUI.deb
+
+RUN mv zerotier-gui-1.3.0.r.-2.noarch.rpm ZeroTier-GUI.rpm
```

Comment:
Any way to not hard code this? Is there any guarantee it will have the same name every time?